VEDA_OS01/models.py

```diff
@@ -731,13 +731,3 @@ def update(self, **fields):
         for attr, value in fields.iteritems():
             setattr(self, attr, value)
         self.save()
-
-    def __unicode__(self):
-        return u'{video} - {provider} - {process_id} - {translation_id} - {lang} - {status}'.format(
-            video=self.video.edx_id,
-            provider=self.provider,
-            process_id=self.process_id,
-            translation_id=self.translation_id,
-            lang=self.lang_code,
-            status=self.status,
-        )
```

Comment:
Let's keep it as it with fewer variables for better debugging.

Reply:
I don't think this will be used now ?

Reply:
@muzaffaryousaf ^^

Reply:
yeah, this won't be used except debugging.